cmake/LibraryConfigurations.cmake

```diff
@@ -58,7 +58,11 @@ if (${PLATFORM} MATCHES "Desktop")
 
 elseif (${PLATFORM} MATCHES "Web")
     set(PLATFORM_CPP "PLATFORM_WEB")
-    set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
+    if(NOT GRAPHICS)
+        set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
+    elseif(${GRAPHICS} MATCHES "GRAPHICS_API_OPENGL_ES3")
+        set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -sFULL_ES3=1")
```

Comment:
For GLFW this is put in target_link_options, rather than CMAKE_EXE_LINKER_FLAGS.
Could you do that instead?

Reply:
Changed it